src/commands/build.ts

```diff
@@ -32,6 +32,21 @@ export class BuildCommand implements Command {
 
   args = [
     {
+      name: 'js-minify',
```

Comment:
discussed in person, our flags will use this format (https://docs.google.com/document/d/1mD1cVhxdxY-bUsGBX44anC1875Afx7UHue4uRCNVc4o)